nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -606,6 +604,11 @@ in {
         # "The files of the app [all apps in /var/lib/nextcloud/apps] were not replaced correctly"
         # Restarting phpfpm on Nextcloud package update fixes these issues (but this is a workaround).
         phpfpm-nextcloud.restartTriggers = [ cfg.package ];
+        phpfpm-nextcloud.serviceConfig.LoadCredential = lib.flatten [
```

Comment:
This `lib.flatten` is unneeded, right?

Reply:
afair lib.optinonal false ... returns [] which are filtered out by flatten

Reply:
Ah, right.

I guess to avoid `lib.flatten` you'd have to use this style: `[ elem1 ] ++ lib.optional cond elem2`

Reply:
yeah but that looked worse IMHO :)